hikari/errors.py

```diff
@@ -87,22 +87,23 @@ class HikariWarning(RuntimeWarning):
     """
 
 
+@attr_extensions.with_copy
 @attr.s(auto_exc=True, slots=True, repr=False, weakref_slot=False)
```

Comment:
Isn't copy inherited here anyway?

Reply:
You were the one that added these to every exception type.

![image](https://user-images.githubusercontent.com/69713762/92336364-b5db5680-f097-11ea-9304-f9bd4f8f04fe.png)

I was just following the consistency you were trying to enforce when you implemented this.

If this isn't valid, can we please open an issue to adjust this, as it isn't in scope of this PR.

Reply:
I just added it to the base exception classes we use